clang/test/CXX/basic/basic.start/basic.start.main/p3.cpp

```diff
@@ -8,6 +8,9 @@
 // RUN: %clang_cc1 -fsyntax-only -verify %s -DTEST8
 // RUN: %clang_cc1 -fsyntax-only -verify %s -DTEST9
 // RUN: %clang_cc1 -fsyntax-only -verify %s -DTEST10 -ffreestanding
+// RUN: %clang_cc1 -fsyntax-only -verify -pedantic-errors %s -DTEST11
+// RUN: %clang_cc1 -fsyntax-only -verify -pedantic-errors %s -DTEST12
+// RUN: %clang_cc1 -fsyntax-only -verify -pedantic-errors %s -DTEST13
```

Comment:
I’d prefer just `-pedantic` or `-Wmain` here so we can test that the warnings are actually warnings as they should be

Reply:
I've changed to `-pedantic`